packages/suite-native-navigation-root/src/dummyScreens/DemoScreen.tsx

```diff
@@ -39,6 +41,12 @@ const typographyItems: TypographyStyle[] = [
     'hint',
     'label',
 ];
+const SELECT_ITEMS: SelectItem[] = [
```

Comment:
nit: I know this is a dummy list of objects. And we still don't have any development conventions established. But I would be happy if we use SCREAMING_SNAKE_CASE just for primitive constants in the future. What is your opinion?

Reply:
screaming definitely just for primitives